en/mulesoft/modules/ROOT/pages/ms_composer_ms_release_notes.adoc

```diff
@@ -1,12 +1,22 @@
 = MuleSoft Composer Release Notes
-:page-revdate: July 12, 2023
+:page-revdate: July 14, 2023
 
 Use MuleSoft Composer to build connected experiences faster. You don't need to wait for development resources to unlock data, integrate apps, and automate business processes that create real customer value.
 
 New versions are automatically available once they are released; you don't need to perform any tasks to move to the new version.
 
 See xref:ms_composer_ms_release_notes_ki.adoc[Known Issues and Limitations for MuleSoft Composer] for a list of known issues and limitations by date.
 
+== July 14, 2023
+
+include::_partials/release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+3]
+
+* In Stripe Connector, in the *On New Refund* trigger now outputs the following fields:
```

Comment:
I'm thinking the second "in" should be removed but not sure if it's intentional.